gows/main.go

```diff
@@ -23,14 +22,17 @@ func main() {
 	}
 }
 
-func handleConnection(w http.ResponseWriter, count int) {
+func handleConnection(w http.ResponseWriter, count int32) {
 	// add 2 second delay to every 10th request
 	if (count % 10) == 0 {
 		println("Adding delay. Count: ", count)
 		time.Sleep(2 * time.Second)
 	}
-	html, _ := ioutil.ReadFile("hello.html") // read html file
-	w.Header().Add("Connection", "keep-alive")
```

Comment:
why is the header removed?

Reply:
net/http set this header automatically if client sent `Connection: keep-alive`. So you don't need to do.